packages/graph-lines/docs/demo/config.js

```diff
@@ -0,0 +1,60 @@
+module.exports = {
+  elements: {
+    'graph-lines': '../..'
+  },
+  models: [
+    {
+      id: '1',
+      element: 'graph-lines',
+      minimumWidth: 500,
+      multiple: false,
+      partialScoring: [],
+      feedback: {
+        correct: {
+          type: 'none',
+          default: 'Correct'
+        },
+        partial: {
+          type: 'none',
+          default: 'Nearly'
+        },
+        incorrect: {
+          type: 'none',
+          default: 'Incorrect'
+        }
+      },
+      model: {
```

Comment:
do you think we can simplify this model? we have decided that we don't need to bind ourselves to the legacy model anymore. an example would be to have just {domain: {}, range: {}} and remove the `model.config` step etc.

Reply:
Alright, will need to re-work the entire component base then though. Make sure it all works correctly, not a small change.

Reply:
As discussed, because of the big implications elsewhere too, we'll postpone this update to a later time.

Details can be found in the issue below.

https://github.com/pie-framework/pie-elements/issues/29